demos/tests/image_sequences.py

```diff
@@ -135,4 +135,17 @@
         image_net_arg('00037128'),
         image_net_arg('00048316'),
     ],
+
+    '500x375x3': [
```

Comment:
Is this just a sequence of arbitrary same-sized images? I don't think we're going to get good code coverage with a sequence like that - you can't reidentify faces if there aren't any faces.

Reply:
Updated images